### PR TITLE
Add the documentation for proxy settings

### PR DESCRIPTION
**Pull request by sebastienrussell**
@utzcoz The documentation for the proxy settings

https://github.com/robolectric/robolectric/pull/7713

### CODE REVIEW

**Comment by sebastienrussell** [comment]
I put what I'm doing in my project but feel free to tell me if you prefer a string instead 😄

File: configuring.md [hunk 1]
```diff
@@ -127,11 +129,15 @@ android {
...
       systemProperty 'robolectric.dependency.repo.username', 'username'
       systemProperty 'robolectric.dependency.repo.password', 'password'
+      systemProperty 'robolectric.dependency.proxy.host', project.findProperty("systemProp.https.proxyHost") ?: System.getenv("ROBOLECTRIC_PROXY_HOST")
+      systemProperty 'robolectric.dependency.proxy.port', project.findProperty("systemProp.https.proxyPort") ?: System.getenv("ROBOLECTRIC_PROXY_PORT")
     }
   }
 }
 ```
 
+
```
Comment (utzcoz) [inline configuring.md:139]: IMO, we don't need space here.

File: configuring.md [hunk 2]
```diff
@@ -113,6 +113,8 @@ Some additional options can be configured globally by setting these system prope
 * **robolectric.dependency.repo.url** — Set the URL of the Maven repository to use for the runtime dependencies (default `https://repo.maven.apache.org/maven2/`).
 * **robolectric.dependency.repo.username** - Username of repository that you defined in **robolectric.dependency.repo.url**.
 * **robolectric.dependency.repo.password** - Password of repository that you defined in **robolectric.dependency.repo.url**.
+* **robolectric.dependency.proxy.host** — Set the host of the proxy to use for the runtime dependencies.
```
Comment (utzcoz) [inline configuring.md:116]: These two properties are only available from Robolectric 4.10, what about adding version requirements for it?

File: configuring.md (hunk 1 above)
Comment (sebastienrussell) [reply configuring.md]: Removed 😄! I was gonna add a section but finally chose otherwise.

File: configuring.md (hunk 2 above)
Comment (sebastienrussell) [reply configuring.md:120]: I forgot about that, but now it's change

**Review by utzcoz — APPROVED** [review]

**Comment by utzcoz** [comment]
@sebastienrussell Thanks!!!. cc @hoisie .

**Pull request merged**